tests/test-sources/example-configurations/issues.nix

```diff
@@ -125,23 +124,23 @@
               end
             '';
           };
-          sources = [
-            { name = "nvim_lsp"; }
-            {
-              name = "luasnip";
-              option = {
-                show_autosnippets = true;
-              };
-            }
-            { name = "path"; }
-            { name = "buffer"; }
-          ];
         };
       };
       barbar.enable = true;
       web-devicons.enable = true;
     };
 
+    # Completion sources
+    cmp-path.enable = true;
+    cmp-buffer.enable = true;
+    cmp-nvim-lsp.enable = true;
+    cmp_luasnip = {
+      enable = true;
+      cmp.default = {
+        option.show_autosnippets = true;
+      };
+    };
+
```

Comment:
These look outside the plugin block

Reply:
`plugins.*.cmp.default` maps to `plugins.cmp.settings.sources [ (it // { inherit (cfg.cmp) name; }) ]`